### PR TITLE
[T2] resources (optional for T2)

### PR DESCRIPTION
**Pull request by zikalino**

### CODE REVIEW

File: specification/resources/resource-manager/Microsoft.Resources/stable/2016-09-01/links.json [hunk 1]
```diff
@@ -386,6 +386,14 @@
...
       "type": "string",
       "description": "The API version to use for the operation."
+    },
+    "ApiVersionParameterExplicit": {
+      "name": "api-version",
+      "in": "query",
+      "required": true,
+      "type": "string",
+      "description": "The API version to use for the operation.",
+      "x-ms-api-version": false
```
Comment (raych1) [inline specification/resources/resource-manager/Microsoft.Resources/stable/2016-09-01/links.json:396]: What's the purpose to add a new api version parameter? What's this extension used for? I didn't find this from [autorest extension document](https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md).

**Comment by raych1** [comment]
@Tiano2017 , can you please review the change on readme?

File: specification/resources/resource-manager/Microsoft.Resources/stable/2016-09-01/links.json (hunk 1 above)
Comment (zikalino) [reply specification/resources/resource-manager/Microsoft.Resources/stable/2016-09-01/links.json]: this is to enable generating explicit api_version parameter autorest.python v5.
in v3 parameter was always generated, in v5 it's not unless i add x-ms-api-version: false
this has no effect on autorest.python v3

**Comment by raych1** [comment]
@Tiano2017 , please review the changes. @zikalino , can you have someone from your team to review the readme changes?

**Comment by raych1** [comment]
@qiaozha , can you please review the readme changes?

**Comment by qiaozha** [comment]
@zikalino This is not ready to merge right? we have to wait for the Track2 release time schedule?

**Comment by raych1** [comment]
@zikalino , are you still working on this PR?

**Comment by raych1** [comment]
Hi @zikalino , I close this PR. Please feel free to re-open it if necessary.

**Comment by JackTn** [comment]
Hi, @zikalino. The PR has be closed for a long time and it's related branch still exist. If no further used for over 14 days, I will delete this related branch. Please tell me if you still need this branch.

**Comment by JackTn** [comment]
Hi, @zikalino. The PR has be closed for a long time and it's related branch still exist. Please tell me if you still need this branch or i will delete it in 14 days.

**Pull request closed without merging**